mycoffee/functions.py

```diff
@@ -89,20 +86,39 @@ def load_params(args):
     return params
 
 
-def coffee_calc(params, digits=3):
+def filter_params(params, digits=3):
     """
-    Calculate coffee.
+    Filter params.
 
     :param params: parameters
     :type params: dict
     :param digits: number of digits up to which the given number is to be rounded
     :type digits: int
+    :return: filtered parameters as dict
+    """
+    params["coffee"] = round(params["coffee"], digits)
+    if is_int(params["coffee"]):
+        params["coffee"] = int(params["coffee"])
+    if is_int(params["water_ratio"]):
+        params["water_ratio"] = int(params["water_ratio"])
+    if is_int(params["coffee_ratio"]):
+        params["coffee_ratio"] = int(params["coffee_ratio"])
+    if is_int(params["water"]):
+        params["water"] = int(params["water"])
+    if len(params["info"]) == 0:
+        params["info"] = "Nothing :)"
```

Comment:
Is it a placeholder or you're going to release it with this?
I guess `None` is better but if you want it to be cool, you can keep it.

Reply:
It's just for fun.